src/core/src/core_logic/PatchInstaller.py

```diff
@@ -95,10 +95,6 @@ def start_installation(self, simulate=False):
         # Combining maintenance
         overall_patch_installation_successful = bool(update_run_successful and not maintenance_window_exceeded)
 
-        if not overall_patch_installation_successful:
```

Comment:
Can you add a comment here stating why it is not marked error at this point?

Reply:
Resolved